app/views/layouts/application.html.haml

```diff
@@ -12,6 +12,12 @@
     /[if lt IE 9]
       <script src='//cdnjs.cloudflare.com/ajax/libs/html5shiv/r29/html5.min.js'>
       </script>
+
+    <script src='//cdn.datatables.net/1.10.13/css/jquery.dataTables.min.css'>
```

Comment:
If you choose to use a CDN, keep this but please use haml syntax. This should help: http://html2haml.herokuapp.com/